lib/dbManager.php

```diff
@@ -27,22 +27,28 @@ function __construct ($dbFilename, $flags = SQLITE3_OPEN_READWRITE) {
     // @param isAdmin: optionnal, specifies if the user is an admin. Defaults to 0. 
     // @return bool : TRUE if no error, FALSE otherwise
     public function addUser ($username, $password, $gauthSecret, $isAdmin = 0) {
+
+		// Prepare variables before the query
+		$passwordHash = hash('sha256',$password);
         
-        // Prepare variables before the query
-        $username = SQLite3::escapeString ($username);
```

Comment:
Oh, OK, so that was that line you were referring to when mentioning the SQLite3 library bug.
So all the commits you're proposing are good with me: I tested the whole branch in a test environment, went through all use-cases and it's seems all right. I'll merge now. Thanks for all those fixes, your time, and your contribution.